Note that Synology NAS may support Virtual Machine Manager, not just Container Manager

ukanuk opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).

  Before submitting your pull request, please verify that you have chosen the correct target branch,
  and that the PR preview looks fine and does not include unrelated changes.
-->
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->

Followup from this forum post showing confusion over available Synology installation method  https://community.home-assistant.io/t/zbt-1-usb-dongle-with-synology-nas-ds923/871931/9?u=mlindstrom

## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

ukanuk committed:
Note that Synology NAS may support Virtual Machine manager, not just Container Manager

Followup from this forum post showing confusion over available Synology installation method 
https://community.home-assistant.io/t/zbt-1-usb-dongle-with-synology-nas-ds923/871931/9?u=mlindstrom

frenck commented on source/_includes/installation/container/alternative.md:
```diff
@@ -1,5 +1,7 @@
 ### Synology NAS
 
+Note: different Synology NAS models support the Container Manager package and/or the Virtual Machine Manager (VMM) package. The Container Manager package may install Home Assistant Container. The Virtual Machine Manager package may install Home Assistant Operating System. For more details on VMM, see [Community Guides: Installation on Synology Virtual Machine Manager](https://community.home-assis ...
```
Let's make this text and positioning more "human". Starting this page with this note, seems a bit out of wack/scope. It should be way later in the text, most certainly after the introduction. Please also check out developer documentation on how we document/make notes in our documentation (as they need to be wrapped in a note block).
../Frenck  

<p>
  <a href="https://www.linkedin.com/in/frenck/"><img src="https://github.com/frenck/frenck/raw/main/images/linkedin.svg" width="18" height="18"></a>
  &nbsp;&nbsp;
  <a href="https://youtube.com/@frenck"><img src="https://github.com/frenck/frenck/raw/main/images/youtube.svg" width="18" height="18"></a>
  &nbsp;&nbsp;
  <a href="https://bsky.app/profile/frenck.social"><img src="https://github.com/frenck/frenck/raw/main/images/bluesky.svg" width="18" height="18"></a>
  &nbsp;&nbsp;
  <a href="https://fosstodon.org/@frenck"><img src="https://github.com/frenck/frenck/raw/main/images/mastodon.svg" width="18" height="18"></a>
  &nbsp;&nbsp;
  <a href="https://www.instagram.com/frenck/"><img src="https://github.com/frenck/frenck/raw/main/images/instagram.svg" width="18" height="18"></a>
  &nbsp;&nbsp;
  <a href="https://www.threads.net/@frenck"><img src="https://github.com/frenck/frenck/raw/main/images/threads.svg" width="18" height="18"></a>
  &nbsp;&nbsp;
  <a href="https://www.facebook.com/frenck.dev/"><img src="https://github.com/frenck/frenck/raw/main/images/facebook.svg" width="18" height="18"></a>
  &nbsp;&nbsp;
  <a href="https://x.com/frenck"><img src="https://github.com/frenck/frenck/raw/main/images/x.svg" width="18" height="18"></a>
  &nbsp;&nbsp;
  <a href="https://www.tiktok.com/@frenck.nl"><img src="https://github.com/frenck/frenck/raw/main/images/tiktok.svg" width="18" height="18"></a>
</p>

Blogging my personal ramblings at <a href="https://frenck.dev">frenck.dev</a>

ukanuk committed:
Update alternative.md

Revisions per frenck: Add `{% note %}` formatting, reposition after the introduction, and make more concise.

frenck commented on source/_includes/installation/container/alternative.md:
```diff
@@ -1,6 +1,11 @@
 Synology with DSM now supports container management through the Container Manager package, allowing you to install Home Assistant without the need for command-line. For details about the package (including compatibility-information, if your NAS is supported), see <https://www.synology.com/en-us/dsm/packages/ContainerManager>
+
+{% note %}
+Some Synology models support Home Assistant Operating System via the Virtual Machine Manager (VMM) package. For more details see the [Community Guide](https://community.home-assistant.io/t/installation-on-synology-virtual-machine-manager/281608), [compatibility information](https://www.synology.com/en-us/dsm/packages/Virtualization), and [Requirements and Limitations](https://kb.synology.com/en-u ...
```
We should not link to community topics for instructions. If there is relevant information we should add to our documentation, we should add that (instead of linking away into discussions).

../Frenck

ukanuk replied:
I agree, but I am not sufficiently experienced and qualified to make those more complete edits. I figured while this is not perfect, perfect should not be the enemy of better, and this is still an improvement. Similar to Wikipedia edits I've made which aren't perfect, but move stuff in the right direction.

As a new user I had no clue that HAOS was an easy option on Synology, and thus started with the Docker container and its limitations. While a link to a community guide is not perfect, it is better than no mention at all of HAOS support.
